### ISSUE TITLE
Array params doesn't works on page load

### ISSUE THREAD

**Issue by max-mykhailenko** [router5/router5#47]
Load url
`site.com/prices/#!/group/1/product/1/?programID=1&params[]=2___1&params[]=3___0`

Code and output
```javascript
console.log('window.location =>', window.location.hash);
// window.location => #!/group/1/product/1/?programID=1&params[]=2___1&params[]=3___0
router.start((err, state) => {
  console.log('state =>', state.params.params); // state => ["3___0"]
  console.log('path =>', state.path); // path => /group/1/product/1/?programID=1&params[]=2___1&params[]=3___0
```

And after router start url hash has been changed to
`site.com/prices/#!/group/1/product/1/?programID=1&params[]=3___0`

**Comment by max-mykhailenko** [comment]
Router params 

```
const router = createRouter(
  [
    {
      name: 'group',
      path: '/group/:groupID',
      children: [
        {
          name: 'product',
          path: '/product/:productID/?programID&params[]',
        },
      ],
    },
  ],
  {
    defaultRoute: 'group',
  }
);
```

**Comment by troch** [comment]
Fixed with v2.0.5

**Comment by max-mykhailenko** [comment]
Nope, doesn't work

**Comment by troch** [comment]
Forgot to build route-node, should be fixed with 2.0.6.

**Comment by max-mykhailenko** [comment]
Thank you!
